packages/fluentui/react-northstar/src/components/Portal/usePortalBox.ts

```diff
@@ -39,7 +37,7 @@ export const usePortalBox = (options: UsePortalBoxOptions): HTMLDivElement => {
   }, [className, element, rtl]);
 
   // This effect should always last as it removes element from HTML tree
-  useIsomorphicLayoutEffect(() => {
+  React.useEffect(() => {
```

Comment:
It's okay to use `useEffect` to remove DOM elements, no reason to use `useLayoutEffect` for this.